lib/voodoo_sms.rb

```diff
@@ -46,6 +46,12 @@ def get_sms(from, to, keyword = '')
     end
   end
 
+  def get_dlr_status(reference_id)
```

Comment:
@tomsabin get_delivery_status would definitely be a better name for a method! But we seem to fully reflect Voodoo API methods in other methods?